docker/test/fuzzer/run-fuzzer.sh

```diff
@@ -247,6 +247,9 @@ quit
     fuzzer_pid=$!
     echo "Fuzzer pid is $fuzzer_pid"
 
+    echo "Attaching gdb to the fuzzer itself"
+    gdb -batch -command script.gdb -p $fuzzer_pid &
```

Comment:
Seems the log message and thus this attach is incorrect:
```
2024-03-01 17:53:23	 process 979 is executing new program: /usr/bin/timeout
2024-03-01 17:53:23	 [Thread debugging using libthread_db enabled]
2024-03-01 17:53:23	 Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
2024-03-01 17:53:23	 [Detaching after fork from child process 995]
2024-03-01 18:17:57	 [Inferior 1 (process 979) exited with code 0322]
```

The PID belongs to `timeout` not to `clickhouse-client`